src/renderers/WebGLRenderTarget.js

```diff
@@ -35,6 +35,10 @@ class WebGLRenderTarget extends EventDispatcher {
 		this.depthBuffer = options.depthBuffer !== undefined ? options.depthBuffer : true;
 		this.stencilBuffer = options.stencilBuffer !== undefined ? options.stencilBuffer : false;
 		this.depthTexture = options.depthTexture !== undefined ? options.depthTexture : null;
+		this.ignoreDepthForMultisampleCopy = options.ignoreDepth !== undefined ? options.ignoreDepth : true;
+		this.hasExternalTextures = false;
```

Comment:
Would it be possible to reduce the number of new properties in `WebGLRenderTarget` but storing more information into the properties object of a render target? I mean this pattern:
```js
const renderTargetProperties = properties.get( renderTarget );
renderTargetProperties.__ hasExternalTextures = true;
```
Evaluating the code, this should also be possible for at least `ignoreDepthForMultisampleCopy`.

Reply:
I think that makes sense for `hasExternalTextures` but not the other values I *think* because they come from the client which might be allocating these render targets before the `properties` object exists.